apps/front/src/components/user/UserCreateForm.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <h1>Register</h1>
+  <form @submit.prevent.stop="registerUser">
+    <UserForm
+      v-model:email="email"
+      v-model:password="password"
+      v-model:password-confirm="passwordConfirm"
+      :is-password-confirmed="isPasswordConfirmed"
```

Comment:
Should passwordConfirm and isPasswordConfirmed really be handled outside the component ?